test/e2e/certificates_test.go

```diff
@@ -354,6 +355,50 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("OCP-65035-CLOUD_CREDENTIALS_SECRET_NAME works well in mint/passthrough clusters", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
```

Comment:
@geliu2016 from the output of either your local test or the PR's bottom's e2e job "ci/prow/e2e-operator", you can see the scenarios' titles are composed by strings in Describe() Context() It():
```
Cert Manager Suite: [It] ACME Certificate dns-01 challenge using ambient credentials should obtain a valid LetsEncrypt certificate
Cert Manager Suite: [It] ACME Certificate OCP-65035-CLOUD_CREDENTIALS_SECRET_NAME works well in mint/passthrough clusters should obtain a valid LetsEncrypt certificate
```
Existing scenario title `ACME Certificate dns-01 challenge using ambient credentials should obtain a valid LetsEncrypt certificate` from the combined Describe() Context() It() strings is a good human-readable title. But your title `ACME Certificate OCP-65035-CLOUD_CREDENTIALS_SECRET_NAME works well in mint/passthrough clusters should obtain a valid LetsEncrypt certificate` from the combined Describe() and your Context() It() strings is a little not much human-readable. May need to update your Context() It() strings to compose a better scenario title.

Reply:
You can BTW therefore update existing scenario title `ACME Certificate dns-01 challenge using ambient credentials should obtain a valid LetsEncrypt certificate` to be `ACME Certificate dns-01 challenge using ambient credentials should obtain a valid LetsEncrypt certificate in AWS non-sts cluster` so that above two scenarios are clear for new readers when they just read the titles.

Reply:
updated.

Reply:
@geliu2016 sorry if I did not say clear. Per above explanation about "combined", it is expected to update as below:
```
	Context("dns-01 challenge using ambient credential", func() {
		It("should obtain a valid LetsEncrypt certificate in AWS non-sts cluster", func() {
... // Original Dev's automated aws case.
		})

		It("should obtain a valid LetsEncrypt certificate in GCP mint/passthrough clusters-65035", func() {
... // Your automated gcp case.
	})
```

Reply:
updated.

Reply:
@geliu2016 As said in Slack, please also update the `It()` string for the AWS scenario as above two comments. Thanks!